src/ducks/account/AccountDetailsPage.jsx

```diff
@@ -6,20 +6,21 @@ import LoanDetailsPage from 'ducks/account/LoanDetailsPage'
 import { isLoanGroup, isReimbursementsVirtualGroup } from 'ducks/groups/helpers'
 import { ACCOUNT_DOCTYPE, GROUP_DOCTYPE } from 'doctypes'
 import { ReimbursementsPage } from 'ducks/reimbursements'
+import flag from 'cozy-flags'
 
 const getComponent = filteringDoc => {
   if (filteringDoc._type === ACCOUNT_DOCTYPE) {
     const accountType = getAccountType(filteringDoc)
 
-    if (accountType === 'Loan') {
+    if (accountType === 'Loan' && flag('loan-details-page')) {
       return LoanDetailsPage
     } else if (accountType === 'Reimbursements') {
       return ReimbursementsPage
     } else {
       return TransactionsPageWithBackButton
     }
   } else if (filteringDoc._type === GROUP_DOCTYPE) {
-    if (isLoanGroup(filteringDoc)) {
+    if (isLoanGroup(filteringDoc) && flag('loan-details-page')) {
```

Comment:
The rationale here being that if the flag is not active, we will show the TransactionsPage which will show "No transactions here" right ?

Reply:
Exactly